custom-connectors/Tulip/readme.md

```diff
@@ -0,0 +1,29 @@
+### Tulip Connector
```

Comment:
@eddya101 Please remove all the custom connector files from this PR. You can submit them with a separate pull request. We will approve and merge it. Thanks.

Reply:
@Amjed-Ayoub All files are now committed. Do you want me to close this pull request and create a new one? Or are you able to approve this one?